templates/base.html

```diff
@@ -36,7 +40,8 @@
 
     <link rel="stylesheet" href="{% static "css/calendar.min.css" %}">
     <script src="{% static "js/calendar.min.js" %}"></script>
-    <script src="{% static "js/luxon.js" %}"></script>
+<!--    <script src="{% static "js/luxon.js" %}"></script> -->
+    <script src="https://cdn.jsdelivr.net/npm/luxon@1.17.2/build/global/luxon.min.js"></script>
```

Comment:
I'm guessing this is a newer version than the file we have with the project? Might be a good idea to just replace the project file with the updated version and still refer to our local copy. Not sure if we have a best practice for this sort of thing yet.